lib/parity/environment.rb

```diff
@@ -100,5 +110,12 @@ def basename
     def pass_through
       [subcommand, arguments].join(' ').strip
     end
+
+    def skip_migration?
+      Kernel.system %{
+        git fetch #{environment} &&
+        git diff --quiet #{environment}/master..master -- db/migrate
```

Comment:
Trying to think through whether there's any edge cases to consider with `production/master..master`. Anything terrible that can go wrong if I haven't fetched from `origin` or something? I think it's fine, just giving it some scrutiny.